### PR TITLE
Kucoinfutures fetchFundingRateHistory

### PR DESCRIPTION
**Pull request by samgermain**
fixes: #16549

I'm not sure where this endpoint comes from, it's talked about on #16549

----------------------

```
% kucoinfutures fetchFundingRateHistory BTC/USDT:USDT undefined 5
2023-01-30T22:33:43.986Z
Node.js: v18.4.0
CCXT v2.6.93
kucoinfutures.fetchFundingRateHistory (BTC/USDT:USDT, , 5)
2023-01-30T22:33:47.152Z iteration 0 passed in 1793 ms

       symbol | fundingRate |     timestamp |                 datetime
----------------------------------------------------------------------
BTC/USDT:USDT |      0.0001 | 1674993600000 | 2023-01-29T12:00:00.000Z
BTC/USDT:USDT |      0.0001 | 1675022400000 | 2023-01-29T20:00:00.000Z
BTC/USDT:USDT |      0.0001 | 1675051200000 | 2023-01-30T04:00:00.000Z
BTC/USDT:USDT |      0.0001 | 1675080000000 | 2023-01-30T12:00:00.000Z
BTC/USDT:USDT |      0.0001 | 1675108800000 | 2023-01-30T20:00:00.000Z
5 objects
2023-01-30T22:33:47.152Z iteration 1 passed in 1793 ms
```

### CODE REVIEW

File: js/kucoinfutures.js [hunk 1]
```diff
@@ -93,6 +93,7 @@ module.exports = class kucoinfutures extends kucoin {
                     'private': 'https://openapi-v2.kucoin.com',
                     'futuresPrivate': 'https://api-futures.kucoin.com',
                     'futuresPublic': 'https://api-futures.kucoin.com',
+                    'webFront': 'https://futures.kucoin.com/_api/web-front',
```
Comment (carlosmiei) [inline js/kucoinfutures.js:96]: @samgermain is this an internal API?

Comment (samgermain) [reply js/kucoinfutures.js:96]: @carlosmiei I really don't don't know, I found out about it from https://github.com/ccxt/ccxt/issues/16549 but that's all I know about it

Comment (carlosmiei) [reply js/kucoinfutures.js:96]: @samgermain I see but I'm pretty sure it is, @kroitor what should we do here?

**Comment by carlosmiei** [comment]
LGTM ✅ 

```
p kucoinfutures fetchFundingRateHistory BTC/USDT:USDT None 2
Python v3.10.9
CCXT v2.6.93
kucoinfutures.fetchFundingRateHistory(BTC/USDT:USDT,None,2)
[{'datetime': '2023-02-02T20:00:00.000Z',
  'fundingRate': 0.0001,
  'info': {'granularity': 28800000,
           'symbol': 'XBTUSDTM',
           'timePoint': 1675368000000,
           'value': 0.0001},
  'symbol': 'BTC/USDT:USDT',
  'timestamp': 1675368000000},
 {'datetime': '2023-02-03T04:00:00.000Z',
  'fundingRate': 0.0001,
  'info': {'granularity': 28800000,
           'symbol': 'XBTUSDTM',
           'timePoint': 1675396800000,
           'value': 0.0001},
  'symbol': 'BTC/USDT:USDT',
  'timestamp': 1675396800000}]
```

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**